scripts/konduit.sh

```diff
@@ -123,6 +124,64 @@ init_setup() {
 
    # Set service ports
    DB_PORT=5432
+
+   # Set variables if using a separate deployment to access the database
+   if [ "${Jumppod}" != "" ]; then
+      OLDINST=${INSTANCE}
+      INSTANCE="konduit-app-${RANDOM}"
+      cat - << EOF | sed s/APPNAME/${INSTANCE}/ > jumppod.yaml
```

Comment:
The heredoc string accepts variables. This could be written a bit simpler:
```shell
      cat - << EOF > jumppod.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: ${INSTANCE}
```

Reply:
updated